### ISSUE TITLE
Eliminate sign warnings for hex numeric literals

### ISSUE THREAD

**Issue by olofson** [olofson/eel#107]
Although EEL actually treats all numeric bases alike - and it supports anything from 2 through 64 (IIRC) - it is common in other languages to assume that hex (and possibly binary and octal) values are "raw", implying that the sign bit is just another bit. EEL, however, will invariably warn if a numeric literal parses into a value that differs in sign from the original literal value. We should probably, perhaps optionally, make exceptions for bases commonly used as "raw" literals.

In short, we don't want warnings like this one:
```
| WARNING: In module './render-sprites.eel' ("./render-sprites.eel"),
| WARNING:   at line 44, in columns 33..58:
| WARNING: Unsigned 32 bit numeric literal 4294967295/0xffffffff parsed as negative 32 bit signed integer.
```
